src/main/java/org/dita/dost/writer/KeyrefPaser.java

```diff
@@ -83,6 +83,7 @@
         ki.add(new KeyrefInfo(TOPIC_DATA, ATTRIBUTE_NAME_HREF, false, true));
         ki.add(new KeyrefInfo(TOPIC_DATA_ABOUT, ATTRIBUTE_NAME_HREF, false, true));
         ki.add(new KeyrefInfo(TOPIC_IMAGE, ATTRIBUTE_NAME_HREF, false, true));
+        ki.add(new KeyrefInfo(SVG_D_SVGREF, ATTRIBUTE_NAME_HREF, false, true));
```

Comment:
Also not sure if this is going to do anything; there is a check for `TOPIC_XREF` a few lines down that should also catch any specialized `<xref>`. But I might be missing something else that is gained by having an entry for `SVG_D_SVGREF`?

Reply:
This is actually needed, because `xref` can contain mixed content, and `svgref` is empty. Thus they need separate element configurations.